packages/juniper_srx/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.10.1-next"
+  changes:
+    - description: Juniper attaches packet size to tcp input
```

Can you make this a description of what the change does?

```suggestion
    - description: Handle Juniper packet size suffix in TCP input.
```